standalone-metastore/src/test/java/org/apache/hadoop/hive/metastore/TestRetriesInRetryingHMSHandler.java

```diff
@@ -24,26 +24,25 @@
 
 import javax.jdo.JDOException;
 
-import org.apache.hadoop.hive.conf.HiveConf;
+import org.apache.hadoop.conf.Configuration;
 import org.apache.hadoop.hive.metastore.api.MetaException;
+import org.apache.hadoop.hive.metastore.conf.MetastoreConf;
+import org.apache.hadoop.hive.metastore.conf.MetastoreConf.ConfVars;
 import org.junit.BeforeClass;
 import org.junit.Test;
 import org.mockito.Mockito;
 
 public class TestRetriesInRetryingHMSHandler {
 
-  private static HiveConf hiveConf;
+  private static Configuration conf;
   private static final int RETRY_ATTEMPTS = 3;
 
   @BeforeClass
   public static void setup() throws IOException {
-    hiveConf = new HiveConf();
-    int port = MetaStoreTestUtils.findFreePort();
```

Comment:
Can you specify we we removed these lines? How does it know which port to start on? This is needed to remove flakiness of this test if I remember correctly other wise it might fail complaining the port is not available.

Reply:
I'm not sure they're necessary, as the test passes without them.  But I can put them back.